### DISCUSSION TITLE
Hub not visible to challenge admin (back-end)

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#483] [Q&A] [answered]
I added a new challenge admin to our Digital Twin challgenge. When she goes in to the admin section and selectes "hub", there are no hubs showing. If I give her a direct link to the Digital Twin admin page, she can however enter and manage it.

**Comment by techsmyth** [comment, marked as answer]
Hi Flori,

This should become a non-issue in the new interface as the settings for the Challenge are shown directly as one of the tabs:

![image](https://user-images.githubusercontent.com/30729240/145675916-f39bfa34-b535-4347-acf3-71e9271c7629.png)

Please let me know if this solves it when the challenge admin is on the new interface...

Cheers, Neil
